ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/geb/AbstractModelCatalogueGebSpec.groovy

```diff
@@ -392,6 +418,14 @@ abstract class AbstractModelCatalogueGebSpec extends GebReportingSpec {
         }
     }
 
+    def descendTree(String... names) {
+        for(String name in names) {
+            noStale({ $('.catalogue-element-treeview-name', text: name) }) {
```

Comment:
should be calling `selectInTree` here. the name `descendInTree` could be misleading as there is no guarantee that the `names` will form a descend path as you're simply clicking to first visible element.